mailpoet/tests/DataFactories/Settings.php

```diff
@@ -181,12 +181,17 @@ public function withWooCommerceEmailCustomizerDisabled() {
   }
 
   public function withDeactivateSubscriberAfter3Months() {
-    $this->settings->set('deactivate_subscriber_after_inactive_days', 90);
+    $this->settings->set('deactivate_subscriber_after_inactive_days', '90');
```

Comment:
I decided to change this and the other `deactivate_subscriber_after_inactive_days` settings to be strings in order to simulate what we would expect to see in the real world. When we `set` something, the value gets put into an in-memory cache that sticks around for the remainder of the request. If we `get` it, we get the in-memory value, which could differ from the value we would get if we had to go to the database .

In other words, keeping these as integers would mean our tests would be getting integers when calling `$this->settings->get('deactivate_subscriber_after_inactive_days')`, but in the vast majority of cases (i.e. any time a value is actually retrieved from the database), it would be a string.